### PR TITLE
Workfiles: keep Browse always enabled

### PR DESCRIPTION
**Pull request by kalisp**
## Changelog Description
Browse might make sense even if there are no workfiles present, actually in that case it makes the most sense (eg. I want to locate workfile from outside - from Desktop for example).

## Testing notes:
1. Browse button in Workfiles should be enabled even in new tasks where no workfiles are present

### CODE REVIEW

**Comment by BigRoy** [comment]
With this change the browse button is always enabled right? So does it still make sense to even keep the line around where it's setting the state?

File: openpype/tools/workfiles/files_widget.py [hunk 1]
```diff
@@ -379,7 +379,7 @@ def _update_asset_task(self):
 
             # Disable/Enable buttons based on available files in model
             has_valid_items = self._workarea_files_model.has_valid_items()
-            self._btn_browse.setEnabled(has_valid_items)
+            self._btn_browse.setEnabled(True)
```
Comment (iLLiCiTiT) [inline openpype/tools/workfiles/files_widget.py:382]:
```suggestion
```

Comment (iLLiCiTiT) [reply openpype/tools/workfiles/files_widget.py:382]: > With this change the browse button is always enabled right? So does it still make sense to even keep the line around where it's setting the state?

I've checked the code and it's true.

**Comment by iLLiCiTiT** [comment]
NOTE: When the workarea does not exist, it will use first existing directory. In extreme cases, when tha workdir path would look like this: `P:/projects/kuba_each_case/Shots/sq01/sq01_sh0010/work/compositing`, it could end up in `P:/projects/kuba_each_case` if `Shots` does not have any folders yet.
- this is probably not an issue, just pointing out

**Review by 64qam — DISMISSED** [review]

**Comment by kalisp** [comment]
Kuba's changes work even on nonexistent folder (it creates it and browse is started from there).

**Review by 64qam — APPROVED** [review]

**Pull request merged**